decentraland/unity-renderer
# fix: ensure user profiles in social features

## What does this PR change?

Fetches the user profile when is not in cache in many social features:
- friend requests
- channel members
- friend list
- friend request list
- direct message list

## How to test the changes?

1. Go to: https://play.decentraland.zone/?explorer-branch=fix/ensure-user-profiles
2. Check that the described social features are working as expected
3. Check that the pagination is working correctly when is possible
4. Check that the chat is working properly: #nearby, #channels, private messages

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
